apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.ts

```diff
@@ -87,9 +91,24 @@ export const getAppealInfo = (theCase: Case): AppealInfo => {
   return appealInfo
 }
 
+export const getIndictmentInfo = (rulingDate?: string): IndictmentInfo => {
+  const indictmentInfo: IndictmentInfo = {}
+
+  if (!rulingDate) {
+    return indictmentInfo
+  }
+
+  const theRulingDate = new Date(rulingDate)
```

Comment:
Please consider using `nowFactory` for theses dates. That makes the code more testable by enabling us to inject custom dates when testing this code.

Reply:
We don't use the nowFactory in our api tests